packages/tokenbank_v1/contracts/modules/transfers/QuotaTransfers.sol

```diff
@@ -258,6 +262,20 @@ contract QuotaTransfers is TransferModule
         emit PendingTxCreated(wallet, pendingTxId, now);
     }
 
+    function getTokenValue(address token, uint amount)
+        private
+        returns (uint value)
+    {
+        if (amount == 0) return 0;
+        value = priceCacheStore.tokenPrice(token, amount);
+        if (value == 0) {
+            value = priceOracle.tokenPrice(token, amount);
```

Comment:
I don't think this is much of a problem probably, but small `amount` values can have significant rounding errors compared to the real price. Maybe `decimals` on a token could be used for some standard amount so this can't be the case, not sure.

Reply:
I think it should be fine. Using 'decimals' will probably complicate things...